dessia_common/document_generator.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+""" Write document file. """
+from typing import List, Tuple
+
+import docx
+
+
+class DocxWriter:
+    """ write a docx file. """
+
+    def __init__(self, filename: str):
+        self.filename = filename
+        self.doc = docx.Document()
+        self.headings = None
+
+    def add_headings(self, headings: List[Tuple[str, int]]) -> 'DocxWriter':
+        """ Add a list of headings to the document. """
+        new_doc = self.doc
```

Comment:
There is no need to add "new_" prefix over `doc` and `writer` variables here. Consider naming them `document` and `writer`